engine/include/systems/physics_system.hpp

```diff
@@ -10,16 +19,21 @@ class PhysicsSystem : public System
     PhysicsSystem(ECS& ecs, PhysicsModule& physicsModule);
     ~PhysicsSystem() = default;
 
-    entt::entity CreatePhysicsEntity();
-    void CreatePhysicsEntity(RigidbodyComponent& rb);
     void AddRigidBody(entt::entity entity, RigidbodyComponent& rigidbody);
+    [[nodiscard]] RayHitInfo ShootRay(const glm::vec3& origin, const glm::vec3& direction, float distance) const;
```

Comment:
Is there a reason this function is not a part of the Physics Module? nothing related to the ECS is here except the return value containing an entt::entity (which can be in the physics module as well)

Reply:
I moved it now to physics module 🫡